.gitignore

```diff
@@ -422,3 +422,4 @@
 /utils/wxrc/wxrc_vc[789].sln
 
 /3rdparty/webview2
+build
```

Comment:
This is incorrect. This directory is needed by wxWidgets and should not be excluded. Please remove it from the ignore file and restore its contents.

Reply:
We are compile with Clang Cmake and vsCode , and after reconfigure and rebuild, We have more then 800 chabges...  
Is "Commint" it correct?